test/tests/functional/pbs_two_mom_hooks_resources_used.py

```diff
@@ -129,7 +129,7 @@ def test_Erecord(self):
         test = []
         test += ['#PBS -N JobEndTest\n']
         test += ['echo Starting test at `date`\n']
-        test += ['sleep 1\n']
+        test += ['sleep 15\n']
```

Comment:
This test also doesn't care about this job script.  I'd remove the job script, but the sleep time is right.  I'd set the sleep time of the job to 15 like you are (but in j1.set_sleep_time())